script.simkl/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.simkl"
+       name="Simkl TV Tracker"
+       version="1.0.0"
+       provider-name="David Davó / Simkl">
+  <requires>
+    <import addon="xbmc.python" version="2.6"/>
+    <import addon="script.module.simplejson"/>
```

Comment:
simplejson isn't really needed as it only had a speed gain with many JSON operations for python 2.6. In Kodi v17 all platforms us python 2.7 and it's no longer needed.